server/mem_agent/engine.py

```
@@ -332,3 +350,656 @@ def _subprocess_entry() -> None:
+
+    Args:
+        session_id: The raw session ID
+
+    Returns:
+        Sanitized session ID safe for use in paths
+
+    Raises:
+        ValueError: If session_id is empty or None
+    """
+    if not session_id:
+        raise ValueError("session_id cannot be empty")
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)